pages/common/where.md

```diff
@@ -0,0 +1,7 @@
+# where
+
+> Reports all known instances of command, including aliases, builtins and executables in path.
```

Comment:
Perhaps `of a command - including` would read better? Also, while you explain PATH to be an environment variable below, you've used it in lowercase here. Perhaps we could shuffle things about and keep it uppercased for consistency?

Reply:
This has been copied directly from manual

Reply:
will change it to easier-to-understand language

Reply:
Thanks! Just comment and @mention me once you're done.

Reply:
We always strive to avoid using verbatim language from the man pages. Otherwise, this will just become a copy of the man page !